project/Dependencies.scala

```diff
@@ -25,6 +25,7 @@ object Dependencies {
   val awsElasticloadbalancing = "com.amazonaws" % "aws-java-sdk-elasticloadbalancing" % awsVersion
   val closureCompiler = "com.google.javascript" % "closure-compiler" % "v20150901"
   val commonsHttpClient = "commons-httpclient" % "commons-httpclient" % "3.1"
+  val commonsLang = "commons-lang" % "commons-lang" % "2.4"
```

Comment:
that's used in a couple of places, forgot where, but it was relying on the transitive dependency brought in from guardian configuration